skins/laika/src/components/pages/donation_form/Name.vue

```diff
@@ -1,82 +1,76 @@
 <template>
 <div>
-	<div v-if="addressType === AddressTypeModel.COMPANY" v-bind:class="[{ invalid: showError.companyName }]">
-		<label for="company-name">{{ $t( 'companyname_label' ) }}</label>
-		<input type="text"
-				id="company-name"
-				name="company"
-				:placeholder="$t( 'companyname_label' )"
-				v-model="formData.companyName.value"
-				@blur="validateInput(formData, 'companyName')">
-		<span v-if="showError.companyName" class="error-text">{{ $t( 'form_companyname_error' )  }}</span>
-	</div>
-	<div v-else-if="addressType === AddressTypeModel.PERSON">
-		<label for="salutation">{{ $t( 'salutation_label' ) }}</label>
+	<div v-if="addressType === AddressTypeModel.PERSON">
```

Comment:
Why swap the if statement?

Reply:
My brain kept being confused because the list goes:

Person
Company
Anon

But the code was:

Company
Person
Nothing

So this just seems more logical. Also, since "Person" is selected by default, it saves an ultra small amount of CPU time because previously, the first `if` was always false.

Reply:
Ok, I'm fine with leaving it this way, but the Address Type order has nothing to do with this component.
If UX decides we should reorder to Anon Company Person you won't go and reorder the `if-else`s here.